lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -59,12 +62,10 @@ class RenderForm extends React.Component {
 
     if (this.validateData(data)) {
       request.open('POST', url, true);
-      request.setRequestHeader('Authorization', `Bearer ${this.props.context.accessToken}`);
-      request.setRequestHeader('Content-Type', 'application/json;charset=UTF-8');
+      request.setRequestHeader('Content-type', 'application/json;charset=UTF-8');
+      request.setRequestHeader('Access-Control-Allow-Origin', '*');
```

Comment:
@dsjen @efischer19 I tried this header because I am getting `Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.` But I am still getting same error.

Why we had added this proxy in studio instead of LMS?
How I can fix this issue ?

Reply:
I'm not sure, but I think you may need to set the `X-CSRF-Token` instead.